docs/admin/deployment.rst

```diff
@@ -99,6 +99,21 @@ you create:
    production.
    Adds some additional django apps that can be helpful during day to day development.
 
+``EMAIL_HOST_PASSWORD``
+   Optional. Password for the SMTP connection (default: ``'apikey'``, no e-mail will be sent if it is set to an empty string).
+
+``EMAIL_HOST_USER``
+   Optional. Username for the SMTP connection (no e-mail will be sent if not defined).
+
+``EMAIL_HOST``
```

Comment:
Sorry for not noticing this earlier: to keep the alphabetic order, this one should be moved in front of `EMAIL_HOST_PASSWORD`.